src/components/Target.vue

```diff
@@ -208,15 +208,9 @@
 <script>
 import _ from 'lodash';
 import $ from 'jquery';
+import { OCSUtil } from 'ocs-component-lib';
 
-import {
-  collapseMixin,
-  sexagesimalRaToDecimal,
-  sexagesimalDecToDecimal,
-  julianToModifiedJulian,
-  decimalRaToSexigesimal,
-  decimalDecToSexigesimal
-} from '@/utils.js';
+import { collapseMixin, sexagesimalRaToDecimal, sexagesimalDecToDecimal, julianToModifiedJulian } from '@/utils.js';
```

Comment:
I'm guessing these will eventually migrate over to OCSUtil in ocs-component-library?

Reply:
That's right - the plan is to move things over as I migrate the components that need them.